runtime/lua/man.lua

```diff
@@ -662,7 +660,8 @@ function M.init_pager()
     vim.cmd.file({ 'man://' .. fn.fnameescape(ref):lower(), mods = { silent = true } })
   end
 
-  set_options(true)
+  vim.g.pager = true
```

Comment:
Although this is a "private" (undocumented) variable, since it's now global we probably should prefix it, as we already do for `g:man_hardwrap` and others.

```suggestion
  vim.g.man_pager = true
```